case-lib/lib.sh

```diff
@@ -343,3 +343,19 @@ disable_kernel_check_point()
 {
     KERNEL_CHECKPOINT="disabled"
 }
+
+is_sof_logger_disabled()
+{
+    # get the firmware type
+    manifest=/etc/sof/manifest.txt
+    FIRMWARE_TYPE=$(grep "firmwaretype" < "$manifest" | awk -F ':' '{print $2}' | sed -e 's/^[ ]*//g;s/\"//g')
+    # disable sof-logger for Zephyr built firmware
+    if [ "$FIRMWARE_TYPE" == "zephyr" ]; then
```

Comment:
This  above seems a bit long and over complicated, could this be shorten into something like this:
```shell
if grep `firmwaretype.*zephyr` /etc/sof/manifest.txt; then
```
Just a really simplified example: you may want a slightly more complex regular expression.
```

Reply:
Good point, I will refine my patch.

Reply:
I didn't know `manifest.txt` was actually a `manifest.json`. The standard, command-line parser for JSON is `jq`, please drop awk, grep etc. and use `jq`:

```
apt-get install jq
jq '.version.firmwaretype' manifest.txt
 => "zephyr"
```

Reply:
Updated. Thanks!